components/publish/PublishLanguagePicker.vue

```diff
@@ -38,6 +52,18 @@ function chooseLanguage(language: string) {
       >
     </div>
     <div max-h-40vh overflow-auto>
+      <template v-if="!languageKeyword.trim()">
```

Comment:
Should this be something like?

```suggestion
      <template v-if="preferredLanguages.length">
```

Reply:
No, my intention was to hide this completely when the user uses the searchbar

Reply:
Ahh, understood 👍